source/iterator/iterator.go

```diff
@@ -114,20 +114,41 @@ type Params struct {
 	BatchSize      int
 }
 
+// HelperObjects returns the helper tables and triggers we use:
+// the snapshot table, the tracking table and the trigger name.
+func (p Params) HelperObjects() (string, string, string) {
```

Comment:
I'm not really happy about this name, I'll try to come up with something better.

Reply:
What's strange to me is that the method is not idempotent, if position is `nil` the result will always be different 😕

Reply:
We may have different understanding and different expectations, but I believe it's not actually expected that this is idempotent. For a given input config (table name etc.), different runs need to create different tracking tables so that they're independent.

Reply:
Updated!